nomad/structs/uuid.go

```diff
@@ -4,4 +4,4 @@ package structs
 // within a single write request. This is to ensure that the Raft message does
 // not become too large. The resulting value corresponds to 0.25MB of IDs or
 // 7282 UUID strings.
-var MaxUUIDsPerWriteRequest = (1024 * 256) / 36
+const MaxUUIDsPerWriteRequest = 7822 // (1024 * 256) / 36
```

Comment:
```suggestion
const MaxUUIDsPerWriteRequest = 7821 // (1024 * 256) / 36
```

Based on the errors in `TestHTTP_EvalsDelete` I guess this was being rounded down?

Reply:
Good catch. The truncating of integer division (to 0!) is why this caused the failure in #16112 in the first place!